src/mod/applications/mod_avmd/mod_avmd.c

```diff
@@ -526,19 +531,17 @@ static void avmd_session_close(avmd_session_t *s) {
 static switch_bool_t avmd_media_bug_init(avmd_session_t *avmd_session) {
 	switch_codec_t          *read_codec;
 	switch_codec_t          *write_codec;
-	switch_core_session_t   *session;
+	switch_core_session_t   *session = avmd_session->session;
 	switch_channel_t        *channel = NULL;
 
-	session = switch_core_session_locate(avmd_session->session_uuid);
-	if (session == NULL) {
-		switch_log_printf(SWITCH_CHANNEL_UUID_LOG(avmd_session->session_uuid), SWITCH_LOG_ERROR, "No FreeSWITCH session assigned!\n");
-		return SWITCH_FALSE;
-	}
+    if (session == NULL) {
```

Comment:
perhaps i'm being too pedantic, but this function is using tabs for indenting, yet here are spaces. also, the rest of the code is a mix of spaces and tabs..